ObjectMapper/Core/Operators.swift

```diff
@@ -285,6 +285,44 @@ public func <- <T: Mappable>(inout left: T!, right: Map) {
 	}
 }
 
+// MARK:- Transforms of Mappable Objects - <T: Mappable>
+
+/// Object conforming to Mappable that have transforms
+public func <- <T: Mappable, Transform: TransformType where Transform.Object == T>(inout left: T, right: (Map, Transform)) {
+	let (map, transform) = right
```

Comment:
```
public func <- <Transform: TransformType where Transform.Object: Mappable>(inout left: Transform.Object, right: (Map, Transform)) 
```
What do you think of this function definition instead? I recently updated the other transform operator function definitions to be more like this (using Transform.Object rather than T).

I don't have much more time to look over this PR today, but from first glance, it looks pretty solid.

On another note, I think I might have to soon split up the Operators file into a few files... its getting a little large.